### ISSUE TITLE
Add 'App' section from Whitelist also to Blacklist

### ISSUE THREAD

**Issue by piu130** [blokadaorg/blokada#93]
I love the new feature to add apps to the whitelist. It would be cool to have it also in the blacklist.

**Comment by kar** [comment]
How would it supposed to work? If a given app is in the whitelist, it means it's excluded from ad blocking altogether. What having it in the blacklist would mean?

**Comment by piu130** [comment]
Block every traffic from the apps in the blacklist.

**Comment by rpeter85** [comment]
Just for clarification, what would be the use case to block every traffic for a given app?
Blokada blocks the traffic from websites those are in the hosts lists you configure, and some apps (games, tools, etc) needs internet access to validate if they are purchased from the Play Store, to get your credentials, etc.

**Comment by piu130** [comment]
Lets say I install a Flashlight app that needs access to my contact list to run. That's a problem because it sends them to a server after. But that wont be a problem if one could block every network traffic for the app. (Stupid example but there are other scenarios)

Sorry but maybe I don't really understand how blokada works I just thought this would be a nice feature.

**Comment by kar** [comment]
Ok now it's clear, it's basically the firewall feature. It is high on the list, so stay tuned:)

**Comment by rpeter85** [comment]
Yeah, that's an outgoing traffic, not incoming. Right now Blokada blocks incoming traffic only.
